src/utils/AppRouter.js

```diff
@@ -2,14 +2,17 @@ import {
   BrowserRouter as Router, Redirect, Route, Switch,
 } from 'react-router-dom';
 import React from 'react';
-import LandingPage from './landing/LandingPage';
-import NotFoundPage from './NotFound';
+import NotFoundPage from '../notfound/NotFound';
+import PrivateRoute from './PrivateRoute';
+import HomePage from '../home-page/Home';
+import LandingPage from '../landing/LandingPage';
```

Comment:
Please pick one naming convention and try to stick with it. You have both camelCase and kebab-case for the directory names.

Reply:
fixed, all directories are now kebab-cased